week-2/G-conditionals-2/exercise-3.js

```diff
@@ -8,7 +8,19 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
+function calculateGrade(mark) {
+  if (mark >= 80) {
+    return "A";
+  }
+  if (mark < 80 && mark >= 60) {
```

Comment:
You don't need to check `mark < 80` here since if the code reaches this line you know that this condition already holds (because anything `<= 80` will already have caused the function to return an `A`.

Reply:
done :)